tools/run_tests/dockerize/docker_run.sh

```diff
@@ -32,13 +35,39 @@ else
   cp -r "$EXTERNAL_GIT_ROOT/$RELATIVE_COPY_PATH"/* "/var/local/git/grpc/$RELATIVE_COPY_PATH"
 fi
 
+# ensure the "reports" directory exists
+mkdir -p reports
+
+# TODO(jtattermusch): this is garbage, remove it.
 $POST_GIT_STEP
```

Comment:
Do you know what script is using this, can we use other mechanism?

Reply:
I think it's our "boringssl at head" and "protobuf at head" scripts. I will remove as a followup in a separate PR.

Reply:
Actually, It was used for gcc4.4 compatibility test (and that test is long gone).
https://github.com/grpc/grpc/pull/5384